src/Platform/Microsoft.Testing.Platform/OutputDevice/TerminalOutputDevice.cs

```diff
@@ -578,7 +573,6 @@ public Task ConsumeAsync(IDataProducer dataProducer, IData value, CancellationTo
 
                 break;
             case TestRequestExecutionTimeInfo testRequestExecutionTimeInfo:
-                _testRequestExecutionTimeInfo = testRequestExecutionTimeInfo;
                 break;
```

Comment:
The whole case is redundant now, I think?

Then, TerminalOutputDevice is no longer doing anything with `TestRequestExecutionTimeInfo` and I guess it can be dropped from the `DataTypesConsumed` array?